src/current/v23.1/debezium.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Migrate and Replicate Data with Debezium
+summary: Use Debezium to migrate data to a CockroachDB cluster.
+toc: true
+docs_area: migrate
+---
+
+[Debezium](https://debezium.io/) is a self-hosted, distributed platform that can be used to read data from a variety of sources and push that data into Kafka. You can use Debezium to [migrate data to CockroachDB](#migrate-data-to-cockroachdb) from an existing, publicly hosted database containing application data.
+
+As of this writing, Debezium supports the following database [sources](https://debezium.io/documentation/reference/stable/connectors/index.html):
+
+- MongoDB
+- MySQL
+- PostgreSQL
+- SQL Server
+- Oracle
+- Db2
+- Cassandra
+- Vitess (incubating)
+- Spanner (incubating)
+- JDBC (incubating)
+
+{{site.data.alerts.callout_info}}
+Migrating with Debezium is only recommended for users already familiar with Kafka. Refer to the [Debezium documentation](https://debezium.io/documentation/reference/stable/architecture.html) for information on how Debezium is deployed with Kafka Connect.
+{{site.data.alerts.end}}
+
+## Before you begin
+
+Complete the following items before using Debezium:
+
+- Ensure you have a secure, [publicly available]({% link cockroachcloud/network-authorization.md %}) CockroachDB cluster running the latest **{{ page.version.version }}** [production release](https://www.cockroachlabs.com/docs/releases/{{ page.version.version }}), and have created a [SQL user]({% link {{ page.version.version }}/security-reference/authorization.md %}#sql-users).
+- Have [Debezium](https://debezium.io/), [Kafka Connect](https://docs.confluent.io/platform/current/connect/index.html), and [Kafka](https://kafka.apache.org/) already set up. This documentation assumes you have already added data from your [source database](https://debezium.io/documentation/reference/stable/connectors/index.html) to a Kafka topic.
+
+## Migrate data to CockroachDB
+
+Once all of the [prerequisite steps](#before-you-begin) are completed, you can use Debezium to migrate data to CockroachDB.
+
+1. To write data from Kafka to CockroachDB, use the Confluent JDBC Sink Connector. First use the following `dockerfile` to create a custom image with the JDBC driver:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    FROM quay.io/debezium/connect:latest
```

Comment:
This looks like a Dockerfile to me .. is that important? I don't think `shell` is the right lexer.